projects/savmswap/index.js

```diff
@@ -1,11 +1,10 @@
 const { uniTvlExport } = require('../helper/unknownTokens')
 
 const factories = {
-    prom: '0x3B73a7eDc9dfE4847a20BcCfEf6Eb1c90439f5C9',
```

Comment:
what is the reason for removing prom here? We would like to continue tracking any old chains for historical accuracy

Reply:
yes please